venvs/make.py

```diff
@@ -58,6 +77,12 @@
     flag_value=True,
     help="create or reuse the global temporary virtualenv",
 )
+@click.option(
+    "--config/--no-config",
+    is_flag=True,
+    default=True,
```

Comment:
I am not sure I like the default... Personally if I want a permanent thing I edit the config file, and if I don't I specifically don't... You think this way is more likely?